src/pybind/mgr/mgr_module.py

```diff
@@ -62,7 +62,10 @@ def emit(self, record):
         else:
             ceph_level = 0
 
-        self._module._ceph_log(ceph_level, self.format(record))
+        record = self.format(record)
+        if isinstance(record, unicode):
+            record = record.encode('utf-8')
+        self._module._ceph_log(ceph_level, record)
```

Comment:
Thanks. This doesn't work because in `format` call, implicit encoding with 'ascii' encoding is done if the input record is a Unicode string. So the record needs to be explicitly encoded first.